general/g.gui/main.c

```diff
@@ -103,6 +108,13 @@ int main(int argc, char *argv[])
 	exit(EXIT_SUCCESS);
     }
 
+    if (workspace_files->answer) {
+		sprintf(progname, "%s/gui/wxpython/core/workspace_files.py", G_gisbase());
```

Comment:
Just a note for the future (not for this PR), we should consider an executable Python module and/or a subcommand interface like grass.benchark has (`python/grass/benchmark/__main__.py`, `python/grass/benchmark/app.py`) instead of these random entry points.